nginx.conf

```diff
@@ -10,7 +10,7 @@ http {
         listen 8080;
         server_name default_server;
 
-        location ~ /enhetsregisteret/api/enheter/(.*) {
+        location ~ /enhetsregisteret/api/(enheter|underenheter)/(.*) {
             default_type application/json;
             alias /mockdata/$1.json;
```

Comment:
blir det ikke bedre med en egen mockdata for underenheter? Nå vil vel både /enheter og /underenheter svare for alle orgnr?

Reply:
Det er strengt tatt ikke noe forskjell på de to. Har du solide grunner for å skille de?

Reply:
Hvis du ikke gjør det så er det vel egentlig ikke vits i å legge det til? Alle i underenheter finnes allerede i /enheter, så koden din vil egentlig aldri bruke /underenheter